gtsam/hybrid/HybridSmoother.cpp

```diff
@@ -27,7 +27,9 @@ namespace gtsam {
 Ordering HybridSmoother::getOrdering(
     const HybridGaussianFactorGraph &newFactors) {
   HybridGaussianFactorGraph factors(hybridBayesNet());
-  factors += newFactors;
+  for(auto& factor: newFactors){
```

Comment:
I don't know why the original didn't work, but you could alternatively do `factors.push_back(newFactors)`. The `push_back` should be much more efficient than looping thanks to `std` magic.

Reply:
Thanks @varunagrawal I wasn't aware, I'll change it to what you suggested.